src/app/api/webhooks/route.ts

```
@@ -0,0 +1,77 @@
+    console.error('Error verifying webhook:', err);
+    return new Response('Error occured', {
+      status: 400,
+    });
+  }
+
+  const eventType = evt.type;
+
+  switch (eventType) {
+    case 'user.created':
+    case 'user.updated':
+      const newUser = evt.data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)